src/System.Private.ServiceModel/tools/scripts/CleanUpHttpsPort.cmd

```diff
@@ -0,0 +1,32 @@
+@if "%_echo%" neq "on" echo off
+setlocal
+
+echo Remove Https configuration
+
+net session > NUL
+if NOT [%_ErrorLevel%]==[0] (
+    set __EXITCODE=%_ErrorLevel%
+    echo Error: this script must be called in an elevated window
+	goto :done
+)
+
+call :RemoveHttps 0.0.0.0:8084
+
+call :RemoveHttps 0.0.0.0:44285
+
+call :RemoveHttps 0.0.0.0:443
+
+goto :done
+
+:RemoveHttps
+netsh http delete sslcert ipport %1
```

Comment:
Again I missed set up _ErrorLevel here and "=". I used build.cmd to test it, somehow msbuild.log does not show any errors. Looks like I need to test each script independently.